### PR TITLE
[PT-137]-UpdateAPIresponses

### PR DESCRIPTION
**Pull request by ItsFaizan**
# [PT-137](https://0xbase.atlassian.net/browse/PT-137?atlOrigin=eyJpIjoiMGI2ZTEyNzY0Yjg5NGRhMmE5NGU2NmQ1ZjQxNmJmYzciLCJwIjoiaiJ9) - API Responses

### API responses from debank, solana and btc populated in table

[PT-137]: https://0xbase.atlassian.net/browse/PT-137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -3,78 +3,69 @@
...
-  const handleAssetSelect = (asset: string | null) => {
-    setSelectedAsset(asset)
-    if (asset) {
-      const sortedData = [...dummyData].sort((a, b) => (a.asset_name === asset ? -1 : b.asset_name === asset ? 1 : 0))
-      setTableData(sortedData)
-    } else {
-      setTableData(dummyData)
-    }
-  }
+export default function Dashboard() {
```
Comment (fuddin95) [inline src/app/dashboard/page.tsx:34]: use arrow syntax

File: src/app/dashboard/page.tsx [hunk 2]
```diff
@@ -3,78 +3,69 @@
...
-          'x-api-key': `${process.env.NEXT_PUBLIC_SOLANA_API_KEY}`,
-        },
-      })
-      setSolanaWalletData(solanaData)
-      console.log(solanaData)
+      const [btcData, solanaData, debankData] = await Promise.all([
+        callApi(`/api/v1/portfolio/btc?addresses=${btcAddress}`, 'GET'),
+        callApi(`/api/v1/portfolio/solana?addresses=${solanaAddress}`, 'GET'),
+        callApi(`/api/v1/portfolio/debank?addresses=${debankAddress}`, 'GET'),
+      ])
```
Comment (fuddin95) [inline src/app/dashboard/page.tsx:50]: weren't we were using the new API to get all portfolio where you only need to send an array of address to one endpoint

File: src/app/dashboard/page.tsx [hunk 3]
```diff
@@ -3,78 +3,69 @@
...
     const fetchData = async () => {
-      const healthData = await callApi('/healthy', 'GET')
-      setHealthStatus(healthData.message)
-
       const btcAddress = '37jKPSmbEGwgfacCr2nayn1wTaqMAbA94Z'
-      const btcData = await callApi(`/api/v1/portfolio/btc/${btcAddress}`, 'GET')
-      setBtcWalletData(btcData)
-      console.log(btcData)
+      const solanaAddress = 'D2maScZPbUquayPseWY4S9AM6Dy6CAadRCzb3ieKqdES'
+      const debankAddress = '0x5124fcC2B3F99F571AD67D075643C743F38f1C34'
```
Comment (fuddin95) [inline src/app/dashboard/page.tsx:43]: why do we have hardcoded addresses

File: src/app/dashboard/page.tsx (hunk 1 above)
Comment (ItsFaizan) [reply src/app/dashboard/page.tsx:34]: This is corrected in the other PR

**Review by fuddin95 — APPROVED** [review]

**Pull request merged**